loki/transform/transform_hoist_variables.py

```diff
@@ -151,8 +146,7 @@ def transform_subroutine(self, routine, **kwargs):
             item.trafo_data[self._key]["to_hoist"] = []
             item.trafo_data[self._key]["hoist_variables"] = []
 
-        calls = [call for call in FindNodes(CallStatement).visit(routine.body) if call.name
-                 not in self.disable]
+        calls = [call for call in FindNodes(CallStatement).visit(routine.body) if call.name]
```

Comment:
should be sufficient to just have `calls = FindNodes(CallStatement).visit(routine.body)`

Reply:
Good catch, fixed now (CI to confirm). Thanks!